### PR TITLE
Fixes dashboard improvement

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
#Fixes dashboard improvement
- [x] Change the everipedia logo on the IQ dashboard.
- [x] Swap the right stats on the dashboard page ( all-time high and all-time low) with total Iq locked and total Hiiq stats
- [x] On the stats page, show stat from iq.wiki (total wikis and edits) and rename to iq.wiki
- [x] On the stats page, change the order of the social links
- [x] Replace uni swap v2 for fraxswap ethereum liquidity on the stats page
- [x] Replace quick swap for pair usdc - iq
- [x] Add also fraxswap on polygon
- [x] Update the APR from a 1-year lock to a 4-year lock
- [x] Update dashboard intro.
- [x] Change 24-hour volume change to 24h Volume
- [x] Change the order of providers on the swap page
- [x] Update right-hand side text on the swap page
- [x] Update the logo on the price chart on the dashboard page
- [x] Update the caption under the title on the IQ bridge
- [x] Change the term lock to stake
- [x] Update the caption under the title on the Lock page
- [x] Change HiIQ Holders to HiIQ Circulating Supply
- [x] Update right sidebar text on the voting page
- [x] Update caption text under the title on the stats page
- [x] Change Raffles to NFT Raffles, IQ Raffles to NFT Raffles
- [x] Update caption text under the title on the raffles page
- [x] Update the caption text on the treasury page.

fixes https://github.com/EveripediaNetwork/issues/issues/839

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
image
@Simplypheyie96 , we will need Crypto.com logo

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell , kindly notify me before working on this branch so I can push my update to avoid conflicts

**Comment by Simplypheyie96** [comment]
@Emmanueldmlr here is the logo: https://www.figma.com/file/wosuJaHeV318k1ON82CZpo/%F0%9F%A7%A0-IQ.WIKI?node-id=8469%3A69220

sorry i totally forgot to post again. Got carried away with work

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell, I'm done with the implementation of the liquidity pairs...
image
Thats what it looks like.

**Comment by Emmanueldmlr** [comment]
make sure you finish up with the pending things ( remove unused codeblock including comments, you can refactor if necessary, and ensure the suitable logo is used) let me know when you are done with these...

**Comment by Emmanueldmlr** [comment]
@kesar I will await the ethplorer API key we talked about.

**Comment by Adebesin-Cell** [comment]
@Softdev1 Please provide icons for liquidity stats.
198828739-9c6c3d61-fa60-4006-96b7-d102253e7cfa

**Comment by Softdev1** [comment]
@Adebesin-Cell  https://frax.imgix.net/tokens/polygon-frax.png

**Review by Simplypheyie96 — APPROVED** [review]
Everything  looks great on the ui! Great Job!

**Comment by Simplypheyie96** [comment]
lp tokens.
figma link: https://www.figma.com/file/wosuJaHeV318k1ON82CZpo/%F0%9F%A7%A0-IQ.WIKI?node-id=10962%3A86710
image

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell , those icons/images are too big, it will be nice if we can reduce their sizes.

File: src/components/lock/IncreaseLockTime.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const IncreaseLockTime = () => {
       0,
     )
     setReceivedAmount(amountToBeRecieved)
-  }, [userTotalIQLocked, lockValue])
+  }, [userTotalIQLocked, lockValue, lockEndDate])
```
Comment (Emmanueldmlr) [inline src/components/lock/IncreaseLockTime.tsx:63]: are you sure that this is not causing unnecessary re-rendering?

File: src/components/lock/LockedDetails.tsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ const LockedDetails = ({
       if (differenceInDays > 0) setDaysDiff(differenceInDays)
       else setDaysDiff(0)
     }
-  }, [lockEndDate])
+  }, [lockEndDate, daysDiff])
```
Comment (Emmanueldmlr) [inline src/components/lock/LockedDetails.tsx:81]: Kindly confirm its not causing unnecessary re-rendering

File: src/utils/stats-data.ts [hunk 3]
```diff
@@ -85,27 +141,18 @@ const getHiIQ = async () => {
...
-        'accept-language':
-          'en-US,en;q=0.9,es;q=0.8,pt;q=0.7,gl;q=0.6,et;q=0.5,ca;q=0.4',
-        'content-type': 'application/json',
-      },
-      body: '{"operationName":"pairs","variables":{"allPairs":["0xd6c783b257e662ca949b441a4fcb08a53fc49914"]},"query":"fragment PairFields on Pair {\\n  id\\n  txCount\\n  token0 {\\n    id\\n    symbol\\n    name\\n    totalLiquidity\\n    derivedETH\\n    __typename\\n  }\\n  token1 {\\n    id\\n    symbol\\n    name\\n    totalLiquidity\\n    derivedETH\\n    __typename\\n  }\\n  reserve0\\n  reserve1\\n  reserveUSD\\n  totalSupply\\n  trackedReserveETH\\n  reserveETH\\n  volumeUSD\\n  untrackedVolumeUSD\\n  token0Price\\n  token1Price\\n  createdAtTimestamp\\n  __typename\\n}\\n\\nquery pairs($allPairs: [Bytes]!) {\\n  pairs(first: 500, where: {id_in: $allPairs}, orderBy: trackedReserveETH, orderDirection: desc) {\\n    ...PairFields\\n    __typename\\n  }\\n}\\n"}',
-      method: 'POST',
-    },
-  )
-  const data = await response.json()
+  // const response = await fetch(
```
Comment (Emmanueldmlr) [inline src/utils/stats-data.ts:152]: remove comment or unused codeblock

File: src/utils/stats-data.ts [hunk 4]
```diff
@@ -116,58 +163,84 @@ const getLPs = async () => {
...
-      headers: {
-        accept: '*/*',
-        'accept-language':
-          'en-US,en;q=0.9,es;q=0.8,pt;q=0.7,gl;q=0.6,et;q=0.5,ca;q=0.4',
-        'content-type': 'application/json',
-      },
-      body: '{"query":"{\\n  pair(id: \\"0x9d45081706102e7aaddd0973268457527722e274\\") {\\n    reserveUSD\\n  }\\n}","variables":null}',
-      method: 'POST',
-    },
+  // const response3 = await fetch(
```
Comment (Emmanueldmlr) [inline src/utils/stats-data.ts:172]: Remove comment or unused codeblock

File: styles/globals.css [hunk 5]
```diff
@@ -14,3 +14,11 @@ a {
 * {
   box-sizing: border-box;
 }
+
```
Comment (Emmanueldmlr) [inline styles/globals.css:17]: Is it possible to not have any style defined here?

File: src/components/lock/LockedDetails.tsx (hunk 2 above)
Comment (Adebesin-Cell) [reply src/components/lock/LockedDetails.tsx:81]: Yeah, I’ll check. I added it to remove the dependency warnings.

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell, The issue about the size is not for the logos...It's for the new images you just added...check the public folder to see the images you added. (public/images/crypto.com.png, public/images/penguin.png, public/images/gnostic.avif)

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell file size

**Review by Emmanueldmlr — APPROVED** [review]

**Pull request merged**